### PR TITLE
Major update on WP-CLI and setup

### PR DESCRIPTION
**Pull request by iruzevic**
Dependant to [Eightshift-libs@8.0.0](https://github.com/infinum/eightshift-libs/releases/tag/8.0.0)

### Changed
- All `Components::getManifest` to `Components::getManifestByDir` method.
- All `Components::outputCssVariables($attributes, $manifest, $unique, $globalManifest)` to `Components::outputCssVariables($attributes, $manifest, $unique)`.
- All `renderPartial` to `render` method.
- Wrapper changes `$this->renderWrapperView` to `echo $renderContent;`.

### CODE REVIEW

**Review by goranalkovic-infinum — APPROVED** [review]

File: blocks/init/src/Blocks/components/copyright/components/copyright-options.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const CopyrightOptions = (attributes) => {
 
 	return (
 		<Toggle
-			label={__('Copyright', 'eightshift-frontend-libs')}
+			label={__('Copyright', '%g_textdomain%')}
```
Comment (piqusy) [inline blocks/init/src/Blocks/components/copyright/components/copyright-options.js:15]: I'm probably missing something but how come this `textdomain` convention is only used in this and `site-footer-options.js` files?

**Review by piqusy — APPROVED** [review]
Nice work 👏🏼 

Left a question

File: blocks/init/src/Blocks/components/copyright/components/copyright-options.js (hunk 1 above)
Comment (kancijan) [reply blocks/init/src/Blocks/components/copyright/components/copyright-options.js:15]: @piqusy what? 😄 
image

Comment (iruzevic) [reply blocks/init/src/Blocks/components/copyright/components/copyright-options.js:15]: this was all done for the other blocks in the previous PR so I fixed the additional changes Kanc did for this block

Comment (piqusy) [reply blocks/init/src/Blocks/components/copyright/components/copyright-options.js:15]: Oh, it seems that I've clicked on a link for a single commit and saw just that small change 😄

File: blocks/init/src/Blocks/custom/featured-content/components/featured-content-options.js [hunk 2]
```diff
@@ -104,7 +104,7 @@ export const FeaturedContentOptions = ({ attributes, setAttributes }) => {
 								});
 							}}
 
-							// placeholder={__('All', 'eightshift-frontend-libs')}
+							// placeholder={__('All', '%g_textdomain%')}
```
Comment (piqusy) [inline blocks/init/src/Blocks/custom/featured-content/components/featured-content-options.js:107]: Should it stay or should it go?

**Review by piqusy — APPROVED** [review]
Nice work 👏🏼 

Left a suggestion

**Review by iobrado — APPROVED** [review]

**Pull request merged**